.circleci/config.base.yml

```diff
@@ -819,23 +806,18 @@ jobs:
           path: ~/repo/packages/amplify-e2e-tests/amplify-e2e-reports
     working_directory: ~/repo
 
-  cleanup_resources_after_e2e_runs:
-    <<: *linux-e2e-executor-large
+  trigger_e2e_workflow_cleanup:
+    <<: *linux-e2e-executor-medium
     steps:
-      - restore_cache:
-          key: amplify-cli-repo-{{ .Branch }}-{{ .Revision }}
-      - restore_cache:
-          key: amplify-cli-yarn-deps-{{ .Branch }}-{{ checksum "yarn.lock" }}
       - run:
-          name: 'Run cleanup script'
+          name: 'Trigger Cleanup for this Workflow'
           command: |
-            cd packages/amplify-e2e-tests
-            yarn clean-e2e-resources workflow ${CIRCLE_WORKFLOW_ID}
-          no_output_timeout: 90m
-      - run: *scan_e2e_test_artifacts
-      - store_artifacts:
-          path: ~/repo/packages/amplify-e2e-tests/amplify-e2e-reports
-    working_directory: ~/repo
+            echo "Triggering cleanup for Workflow $CIRCLE_WORKFLOW_ID"
+            curl --request POST \
+              --url https://circleci.com/api/v2/project/gh/aws-amplify/amplify-cli/pipeline \
+              --header "Circle-Token: $AMPLIFY_CLI_CIRCLECI_TRIGGER_TOKEN" \
+              --header "content-type: application/json" \
+              --data '{"branch":"'$CIRCLE_BRANCH'","parameters":{"e2e_workflow_cleanup":true,"setup":false,"e2e_workflow_cleanup_workflow_id":"'$CIRCLE_WORKFLOW_ID'"}}'
```

Comment:
could you check if this step fails if HTTP Response is not successful ?

Reply:
I can follow up with different behaviors on this; I don't think it should be blocking or fail the build, since this step is kind of an after-effect, but we should be alerted to fix this issue somehow without marking the builds as 'failed'. I can do a follow up

Reply:
here is what happens if you get an error response (forced it)
![image](https://user-images.githubusercontent.com/110861985/206819779-3456b362-4f05-41a9-b1cc-a2f51df9b1bb.png)

Reply:
fwiw, this should only fail if circleci is having issues or if their API changes, but they have versioned APIs so we should be good

Reply:
the reason I'm asking is that we should be warned if this integration stops working for some reason.
looks like `--fail-with-body` might be cheap solution to this problem.
see https://superuser.com/questions/590099/can-i-make-curl-fail-with-an-exitcode-different-than-0-if-the-http-status-code-i